src/case-management/repositories/case-repository.js

```diff
@@ -0,0 +1,49 @@
+import { wreck } from '../../server/common/helpers/wreck.js'
+import { Case } from '../entities/case.js'
+import {
+  CaseRepositoryError,
+  CaseNotFoundError,
+  CaseUpdateError
+} from '../errors/case-errors.js'
+
+const toCase = (data) =>
```

Comment:
could we destucture the objects here like so 

` const { payload = {}, ...rest } = data;`